src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java

```diff
@@ -874,6 +877,21 @@ public Message createMessage(JSONObject jsonObject, MessageChannel chan, boolean
         final List<MessageEmbed>       embeds      = map(jsonObject, "embeds",      this::createMessageEmbed);
         final List<MessageReaction>    reactions   = map(jsonObject, "reactions",   (obj) -> createMessageReaction(chan, id, obj));
 
+        if (!jsonObject.isNull("activity"))
+        {
+            JSONObject activityData = jsonObject.getJSONObject("activity");
+            Message.ActivityType activityType = Message.ActivityType.fromId(activityData.getInt("type"));
+            if (activityType == Message.ActivityType.PARTY)
+            {
+                activity = new Message.Activity(activityType, activityData.getString("party_id"));
+            }
+            else
+            {
+                WebSocketClient.LOG.debug("Received an unknown activity type in a message. JSON: {}", jsonObject);
+                activity=  new Message.Activity(activityType, "");
```

Comment:
This equal seems a bit off 😄

Reply:
Does not matter i use switch now and already noticed it

Reply:
I don't think logging the entire message json makes much sense here

Reply:
Alright! I will change that.